packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1320,7 +1320,12 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
       forcePixels(actualNewPixels);
       didUpdateScrollPositionBy(offset);
     }
-    return delta + offset;
+
+    final double result = delta + offset;
+    if (result > -0.000001 && result < 0.000001) {
```

Comment:
Not sure if it can be used here, maybe you can rely on `precisionErrorTolerance`:

```suggestion
    if (result.abs() < precisionErrorTolerance) {
```

Reply:
I was hoping there was something like `precisionErrorTolerance`, thank you!